### DISCUSSION TITLE
Yocto recipe?

### DISCUSSION THREAD

**Discussion by gbe302** [Haivision/srt#1985] [Q&A] [answered]
Has anyone created a Yocto recipe yet?  We are working on a Petalinux project and it would be very helpful to have a Yocto recipe for this library so we can compile it into GStreamer

**Comment by maxsharabayko** [comment, marked as answer]
I don't think there is a Yocto recipe for SRT. Would you consider using [vcpkg](https://github.com/microsoft/vcpkg)? It has a recipe for SRT library. Not sure if it supports Peralinux though.
